### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DanielSzczucki**

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (Sumick) [inline .vscode/tasks.json]: **nitpick:** You should not keep VSC related files in repository, as not every developer that you will work will have the same IDE. Configuration should not be IDE specific.

File: package.json [hunk 2]
```diff
@@ -7,6 +7,17 @@
   "author": "Nerdbord",
   "license": "MIT",
   "scripts": {
-    "server": "node index.js"
+    "server": "node index.ts",
+    "startDev": "tsnd index.ts",
```
Comment (Sumick) [inline package.json]: **issue:** Same here - if I clone your repo, I won't be able to run it, as it's configured for VSC specifically. `ts-node` will be totally enough for that purpose

**Review by Sumick — CHANGES_REQUESTED** [review]
We should get rid of VSC specific configuration, as it restricts the way of how we can work in this repository. Not every developer that you will work with will use the same IDE, so we should set IDE agnostic.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
Thank you for providing me with the code changes and task requirements. Based on the provided changes, I have the following feedback:

### Code Changes
1. The first code change removes a console log statement from the codebase.
2. The second code change adds necessary imports and sets up an express server with middleware to parse JSON requests. It also uses environment variables to set the port number.
3. The third code change updates the `package.json` file to include necessary dependencies and devDependencies.
4. The fourth code change adds a `tsconfig.json` file to the project.

### Task Requirements
1. The task requires setting up Typescript in the project, which has been achieved by adding a `tsconfig.json` file and updating the `package.json` file with necessary dependencies.
2. The task also requires that it should be possible to write `.ts` files and existing `.js` files should be refactored to `.ts`. However, I did not find any changes in the provided code that indicate refactoring of existing `.js` files to `.ts`. 

Overall, the provided changes seem to have achieved the goal of setting up Typescript in the project. However, it is important to refactor existing `.js` files to `.ts` as well to fully utilize Typescript's benefits.

**Review by NerdbordMentor — APPROVED** [review]
Based on the task requirements, the changes made to the code are appropriate. The changes include:

1. Adding the `dotenv` package to load environment variables from a `.env` file.
2. Importing the `express` package and using it to create an instance of the application.
3. Adding middleware to parse incoming requests with JSON payloads.
4. Updating the `scripts` section in `package.json` to use `ts-node` to run the application.
5. Adding `ts-node`, `ts-node-dev`, `@types/express`, `dotenv`, and `express` packages as dependencies.
6. Adding a `tsconfig.json` file to configure TypeScript compilation options.

Overall, the changes made are appropriate for setting up TypeScript in the project. However, there are a few minor issues that could be improved:

1. The `skipLibCheck` option in `tsconfig.json` is set to `true`. While this can improve compilation times, it can also lead to type errors being missed. It's recommended to set this option to `false` for better type checking.
2. The `esModuleInterop` option in `tsconfig.json` is set to `true`. While this can make it easier to import CommonJS modules, it can also lead to unexpected behavior if not used correctly. It's recommended to set this option to `false` and use named imports instead.
3. The `experimentalDecorators` and `emitDecoratorMetadata` options in `tsconfig.json` are enabled. While these options can be useful for working with decorators, they are not required for basic TypeScript usage and can add unnecessary complexity. It's recommended to disable these options unless they are specifically needed.

Other than these minor issues, the changes made are appropriate for setting up TypeScript in the project.

**Pull request merged**